include/rocksdb/cache.h

```diff
@@ -142,6 +143,13 @@ class Cache {
   Cache(const Cache&) = delete;
   Cache& operator=(const Cache&) = delete;
 
+  // Creates a new Cache based on the input value string and returns the result
+  // The value might be an ID, and ID with properties, or an old-style cache
+  // initialization string.
+  static Status CreateFromString(const ConfigOptions& config_options,
```

Comment:
Also give an example of the string value here to help understand how it can be used in other situations.